IAMOTZ/ievents
# #153046250 Set up coveralls

#### What does this PR do?
Set up coveralls for code coverage reporting
#### Description of Task to be completed?
Set up coveralls 
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- run `npm test`
- It should report the coverage locally 
#### Any background context you want to provide?
I used nyc for my code coverage reporting
#### What are the relevant pivotal tracker stories?
#153046250
#### Screenshots (if appropriate)
None
#### Questions:
None
